lib/src/reducers/helices_reducer.dart

```diff
@@ -548,7 +549,7 @@ BuiltMap<int, Helix> helix_position_set_reducer(
 /// are changing in the action, then state will not yet have been updated to the new value; in that case
 /// assign them explicitly in the parameter list; e.g., see [invert_xy_set_helices_reducer]
```

Comment:
Can you also adjust the comment above this function to mention `groups` as well?

Reply:
More generally, can we document what seems to be true, that this function is a wrapper for `util.helices_assign_svg`, which allows one to avoid listing the various parameters that function needs, if they can simply be found in a single instance of `AppState`, but that these optional parameters are necessary when some parts of the `AppState` instance aren't equal to what is needed in `util.helices_assign_svg`? Can you also add a comment above `util.helices_assign_svg` indicating the same (it has a long parameter list; if those parameters can be found in an instance of `AppState` then call `reassign_svg_positions` instead).